src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -840,23 +843,27 @@ public GHPermissionType getPermission(GHUser u) throws IOException {
      *
      * @param users
      *            the users
+     * @param perm
+     *            the permission level
      * @throws IOException
      *             the io exception
      */
-    public void addCollaborators(GHUser... users) throws IOException {
-        addCollaborators(asList(users));
+    public void addCollaborators(GHOrganization.Permission perm, GHUser... users) throws IOException {
```

Comment:
It's fine to add new methods with new parameters, but we need to keep the original methods as well.  

If you believe the old ones are no longer valid you can mark them as deprecated.